### DISCUSSION TITLE
Code Snippet Not Working in TSMl-ui

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#904] [Q&A] [answered]
The following snippet from https://github.com/code4recovery/12-step-meeting-list/wiki/Frequently-Asked-Questions#Can_I_see_types_in_the_meeting_list_And_can_I_adjust_the_Men_and_Women_after_the_meeting_name that I added to my child's functions.php only works on the legacy version. Is there an equivalent for the UI version?

if (function_exists('tsml_custom_flags')) {
		tsml_custom_flags(array('M', 'W', 'O', 'C'));
	}

**Comment by joshreisner** [comment, marked as answer]
oops no, i'll create a ticket

**Reply by joshreisner** [reply]
https://github.com/code4recovery/12-step-meeting-list/issues/905

**Reply by joshreisner** [reply]
#906 pull request created

**Reply by Grateful-Viv** [reply]
This works now, @joshreisner 

Thank you so much.

Since there's no reason to add open, I did amend it slightly
```
if (function_exists('tsml_custom_flags')) {
tsml_custom_flags(array('M', 'W', 'C'));
}
```
